src/org/etools/j1939_84/bus/j1939/packets/model/SpnDefinition.java

```diff
@@ -4,9 +4,20 @@
 package org.etools.j1939_84.bus.j1939.packets.model;
 
 public class SpnDefinition {
-    public int slotNumber;
-    public int spnId;
-    public String spnLabel;
-    public int startBit;
-    public int startByte;
+
+    final public String label;
```

Comment:
Do you think Eric's ok with with direct access of members or should we use getters?

Reply:
I'm fine with final publics.